finalPrep/annotations.js

```diff
@@ -2,41 +2,54 @@
 // Use the double // to create a new comment
 
 //Build a Bear
+//Invoke the function buildABear and pass it 5 arguments, name, age, fur, clothes, and special power.
 function buildABear(name, age, fur, clothes, specialPower) {
+//Declare a variable called greeting and assign it the value of a string containing an interpolation for the 
+//bear's name with a greeting and a request to be it's friend. 
   var greeting = `Hey partner! My name is ${name} - will you be my friend?!`;
+  //Declare a variable called demographics and set it equal to an array, and set elments to name and age. 
   var demographics = [name, age];
+  //Declare a variable called powerSaying that concatenates a string containing a question  "Did you know that I can" and its specialPower. 
   var powerSaying = "Did you know that I can " + specialPower + " ?";
-  var builtBear = {
+  //Declare a variable called builtBear that contains an Object Literal that holds Build A Bear attributes and assign the specifics to those.
+    var builtBear = { 
     basicInfo: demographics,
     clothes: clothes,
     exterior: fur,
     cost: 49.99,
     sayings: [greeting, powerSaying, "Goodnight my friend!"],
     isCuddly: true,
   };
-
+//Log the value of of the object
   return builtBear
 }
-
+//Manipulate 2 object instances of the of buildABear containing the 5 arguments
```

Comment:
See note on line 5 😊

Object Instances are created from classes; `buildABear` is just a regular function!